applications/Liberland_judiciary.md

```diff
@@ -0,0 +1,533 @@
+2. [**Stanford Journal of Blockchain Law & Policy**](https://stanford-jblp.pubpub.org/): This journal publishes articles and essays related to blockchain law and policy. Some recent articles include:
+
+3. [**On-Chain DAO Governance and Decentralized Regulation**](https://www.ssrn.com/index.cfm/en/maxplancklawrps/max-planck-luxembourg-res/) by Max Planck Institute Luxembourg for Procedural Law.
+
+4. [**Cryptocurrency and the Family Courts - Some International Experiences**](https://financialremediesjournal.com/content/cryptocurrency-and-the-family-courts-some-international-experiences.4fc550f56143411ca515055b2ce2872d.htm) – by Financial Remedies Journal
+This article discusses how cryptocurrency is being dealt with in family court cases in England.,
+- **links to your research diary, blog posts, articles, forum discussions or open GitHub issues:** [development JIRA](https://liberland.atlassian.net/jira/software/projects/BLOCKCHAIN/boards/2),
+- **contact with Web3 Foundation:** conversation with Seraya Takahashi from 26-07-2023
+
+- ### **Links**
+[Github](https://github.com/liberland)
+[Docs](https://liberland-1.gitbook.io/wiki/v/public-documents/blockchain/)
+[Testing guide](https://docs.google.com/document/d/1ntrT6bafTD2LfXUG9QLxOieogXELn9icN1y__EGfZ98/edit#heading=h.yfda4shb4ggs)
+[Blockchain website](https://liberland.org/blockchain)
+[Project website](https://liberland.org)
+[Mainnet explorer](https://polkadot.js.org/apps/?rpc=wss%253A%252F%252Fmainnet.liberland.org#/)
+[Mainnet frontend](https://blockchain.liberland.org/signin)
+[Testnet explorer](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ftestchain.liberland.org%2F#/explorer)
+[Testnet frontend](https://testnet.liberland.org)
+[Metaverse](https://liberverse.net/)
+[Liberland Marketplace](https://market.ll.land/)
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 2 months
+- **Full-Time Equivalent (FTE):**  4 FTE
+- **Total Costs:** 30 000 USD.
+
+### Milestone 1 Simple Process
+
+- **Estimated duration:** 1 month
+- **FTE:**  1,5
+- **Costs:** 15,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | MIT |
+| **0b.** | Documentation |  **Inline documentation** of the code and a **tutorial** to spin up a Liberland nodes and test how the new functionality works. |
+| **0c.** | Testing and Testing Guide | Unit tests and testing guide. |
+| **0d.** | Docker | Dockerfile(s) that can be used to test the functionality in this milestone. |
+| 0e. | Article | Publish an **article**/workshop that explains the functioning of the judiciary and its impact on the Polkadot environment |
+| 1. | Substrate: Simple judiciary | Traditional plaintext contracts on chain that can be signed by parties and registered judges |
+| 2. | Substrate: Hybrid contracts | Expanding smart contract solutions to support plaintext clauses, courts as oracles, work with other pallets such as LLM |
```

Comment:
How does plaintext clauses support exactly look like? You store plaintext on-chain, what else is needed?

Reply:
Yeah, basically.

Store them as one part (clause) of a contract, and a way to write/parse in a user friendly way that can be recognizeable as a standard contract. Most of the work for this is in the [contract builder](https://docs.google.com/document/d/1EwgnuHso_6CUJ0mpIm-SZ_Rg_mQmKO1thc4AYHy_lYI/edit#heading=h.un1glcsq82x0).